servers/physics_3d/godot_collision_solver_3d_sat.cpp

```diff
@@ -2114,6 +2116,19 @@ static void _collision_convex_polygon_convex_polygon(const GodotShape3D *p_a, co
 	const Vector3 *vertices_B = mesh_B.vertices.ptr();
 	int vertex_count_B = mesh_B.vertices.size();
 
+	if (edge_count_A * edge_count_B >= EDGE_PAIR_GJKEPA_THRESHOLD) {
+		GodotCollisionSolver3D::CallbackResult callback = SeparatorAxisTest<GodotCapsuleShape3D, GodotCylinderShape3D, withMargin>::test_contact_points;
```

Comment:
The template arguments should both be `GodotConvexPolygonShape3D`.

Reply:
Oh thanks, I am surprised how it even worked, heh.